mne_neuron/dipole.py

```diff
@@ -14,55 +14,109 @@ def _hammfilt(x, winsz):
     return convolve(x, win, 'same')
 
 
-def simulate_dipole(net, n_jobs=1):
-    """Simulate a dipole given the experiment parameters.
+def rmse (a1, a2):
+    from numpy import sqrt
+
+    # return root mean squared error between a1, a2; assumes same lengths, sampling rates
+    len1,len2 = len(a1),len(a2)
+    sz = min(len1,len2)
```

Comment:
hmm ... why not raise a ValueError if the lengths are not equal?

Reply:
If experimental data is longer than simulated data, comparison between the common time points should still be valid

Reply:
But how will you know what are the common time points?

Reply:
My assumption for this code was that time 0 is common. However, I have a new use case for calculating RMSE for an arbitrary range, which I think is closer to your assumption in the comment?

Reply:
umm ... it just seems dangerous to do it this way and could lead to bugs. One way to fix it would be to add a `times` attribute to the `Dipole` object and pass that object to this function instead of a numpy array (or list? the function comes with no documentation so I can't tell when reading the code ...)